popde/density_estimate.py

```diff
@@ -1,3 +1,71 @@
+import numpy as np
+
+class SimpleKernelDensityEstimation:
+    """
+    Fit and evaluate multi-dimensional Kernel Density Estimation (KDE) 
+
+    Methods:
+    --------
+    check_dimensionality():
+        check if the data matches the dimensions of KDE.
+
+    evaluate(points):
+        Evaluate the KDE at given data points.
+    """
+    def __init__(self, data, bandwidth=1.0, kernel='gaussian', dim_names=None):
```

Comment:
I am not sure why you have the `kernel` kwarg, it is not documented and does not do anything. Probably best to remove it

Reply:
removed